Anagram.java

```diff
@@ -28,22 +28,63 @@ public static void main(String args[]) {
 
 	// Returns true if the two given strings are anagrams, false otherwise.
 	public static boolean isAnagram(String str1, String str2) {
-		// Replace the following statement with your code
-		return false;
+		String new1 = preProcess(str1);
+		String new2 = preProcess(str2);
+		char c;
+		for (int i = 0; i < new1.length(); i++) {
+			String cut = "";
+			int indexStr2;
+			c = new1.charAt(i);
+			indexStr2 = new2.indexOf(c);
```

Comment:
1. you dont use `c` outside the loop. better to put inside
2. don't declare without initial value
  instead of lines 33, 36-38, you can write
```
  char c = new1.charAt(i);
  int indexStr2 = new2.indexOf(c);
```